examples/resources/datadog_dashboard_list/resource.tf

```diff
@@ -0,0 +1,60 @@
+# Create a new Dashboard List with two Dashboards
```

Comment:
nit: can we fix the formatting and spacing in there ?

Reply:
I ran a `terraform fmt` on this file and committed the change.